config/webhook/manifests.yaml

```diff
@@ -92,23 +92,3 @@ webhooks:
     resources:
     - cloudstackmachines
   sideEffects: None
-- admissionReviewVersions:
```

Comment:
was this unnecessary?

Reply:
seems so. this file is changed every time when run `make` command.